Readme.md

```diff
@@ -66,8 +66,17 @@ The main goals of this environment are:
         * [Supermicro A2SDV-8C-LN8F](documentation/devices/servers/Supermicro_A2SDV-8C-LN8F.md)
 * Other documentation
   * SONiC
-    * [FAQ for SONiC](documentation/sonic/FAQ_SONiC.md)
-    * [SONiC Testing Environment](documentation/sonic/SONiC_Testing.md)
+      * [General FAQ](documentation/sonic/FAQ.md)
```

Comment:
Added references to the main page, probably you have better descriptions.

Reply:
LGTM, thank you!